frontstage_api/controllers/iac_controller.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+
+from structlog import wrap_logger
+
+from frontstage_api import app
+from frontstage_api.common.request_handler import request_handler
+from frontstage_api.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_iac_from_enrolment(enrolment_code):
+    logger.debug('Retrieving IAC', enrolment_code=enrolment_code)
+    url = app.config['RM_IAC_GET'].format(enrolment_code)
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code == 404:
```

Comment:
I think all `request_handler` requests expect a 2XX response code and return an ApiError wrapping the status. Could all of these cases be handled in the `request_handler`?

Reply:
We could raise the exception from request handler which would save some code for sure.

The downside would be that we get less detailed logging, right now we can log things like, `Email already in use`, `Enrolment code does not exist`
etc. We lose that ability if we just return the error from the handler